tests/unit/test_task7.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from rosyk_task7 import Point, Vector
+
+
+def test_vector_init():
+    vector = Vector(Point(0, 0), Point(5, 8))
+    assert vector.length == pytest.approx(9.433, 0.001)
+    assert vector.x == 5
+    assert vector.y == 8
+
+
+@pytest.mark.parametrize('vectors, expected',
+                         [([Vector(Point(0, 0), Point(5, 8)), Vector(Point(9, 1), Point(6, 3))],
+                           {'x': 2, 'y': 10}),
+                          ([Vector(Point(0, 0), Point(0, 0)), Vector(Point(0, 0), Point(0, 0))],
+                           {'x': 0, 'y': 0})])
```

Comment:
some of your point are just duplications.
move them to constants or to a fixture